#!/usr/bin/env python
PACKAGE = "hexapod_gait_12dof"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


gen.add("TC_Left_amp", double_t, 0, "A double parameter", 0.5, 0.0, 0.5)
gen.add("TC_Right_amp", double_t, 0, "A double parameter", 0.5, 0.0, 0.5)
gen.add("CTr_amp", double_t, 0, "A double parameter", 0.75, 0.0, 0.75)
gen.add("CPG_period", double_t, 0, "A double parameter", 4.0, 0.5, 4.0)
gen.add("Walk", bool_t, 0, "A bool parameter", True)
gen.add("Direction", bool_t, 0, "A bool parameter", True)
gen.add("Arm",bool_t, 0,"Arm/Disarm flight",False)
gen.add("Propellers",int_t,0,"Set proppeller speed",1000,1000,2000)

exit(gen.generate(PACKAGE, "hexapod_gait_12dof", "CPG"))
